dir.props

```diff
@@ -67,6 +67,9 @@
     <PackagesBinDir>$(__PackagesBinDir)</PackagesBinDir>
     <PackagesBinDir Condition="'$(__PackagesBinDir)'==''">$(BinDir).nuget\</PackagesBinDir>
 
+    <PackageInstallPath>$(__PackageInstallPath)</PackageInstallPath>
```

Comment:
It looks like dir.props is the most reasonable place to set that property - the lowest level file that all the .pkgproj files import is the root-level dir.props (setting it in packages.builds did not work). I'll put up a PR to do so.